packages/common/src/store/solana/sagas.ts

```diff
@@ -1,12 +1,11 @@
-import { solanaActions, getContext } from '@audius/common'
-import { put, call, take } from 'typed-redux-saga'
+import { put, call } from 'typed-redux-saga'
 
-import * as backendActions from 'common/store/backend/actions'
-const { setFeePayer } = solanaActions
+import { getContext } from '../effects'
+
+import { setFeePayer } from './slice'
 
 function* watchForFeePayer() {
   const audiusBackendInstance = yield* getContext('audiusBackendInstance')
-  yield take(backendActions.SETUP_BACKEND_SUCCEEDED)
```

Comment:
do we need `waitForRead` or `waitForWrite` or something in this place?

Reply:
nah. getRandomFeePayer has a waitForLibs already, this was unnecessary